admin/class-yoast-form.php

```diff
@@ -244,10 +244,16 @@ public function legend( $text, $attr ) {
 	 * @param string $var        The variable within the option to create the checkbox for.
 	 * @param string $label      The label to show for the variable.
 	 * @param bool   $label_left Whether the label should be left (true) or right (false).
+	 * @param array  $attr       Extra attributes to add to the checkbox.
 	 */
-	public function checkbox( $var, $label, $label_left = false ) {
+	public function checkbox( $var, $label, $label_left = false, $attr = [] ) {
 		$val = $this->get_field_value( $var, false );
 
+		$defaults = [
+			'disabled' => false,
+		];
+		$attr     = wp_parse_args( $attr, $defaults );
+
```

Comment:
I'm noticing a fair amount of duplication of this and

```
$disabled_attribute = disabled(
	$this->is_control_disabled( $var ) || ( isset( $attr['disabled'] ) && $attr['disabled'] ),
	true,
	false
);
```

in this PR. I'd suggest creating a simple method that returns the disabled attribute based on the defaults and whatever set of attributes are passed along.